packages/runtimes/js/src/bundle-url.js

```diff
@@ -1,5 +1,3 @@
-/* globals document:readonly */
-
 var bundleURL = null;
 function getBundleURLCached() {
```

Comment:
The original bugfix got lost somewhere along the way: [`4237582` (#5916)](https://github.com/parcel-bundler/parcel/pull/5916/commits/4237582f322c48b2d55484844fcbbe8a13dd7ead#diff-39112519f42223a8e331d0dac27d01f8a70db6fe3d515ad8016856931a6ccbda)